docs/js/contribution.md

```diff
@@ -0,0 +1,173 @@
+---
+id: contribution-model-sap-cloud-sdk-for-javascript-typescript
+title: Contribution model
+sidebar_label: Contribution model
+description:
+keywords:
+- sap
+- cloud
+- sdk
+- cloud native
+- cloud sdk
+- sap cloud sdk
+---
+
+## Introduction
+
+SAP Cloud SDK for JavaScript welcomes contributions that bring value to SDK users by simplifying and/or improving cloud application development within SAP ecosystem. The contributed feature or API library becomes a part of SDK and is released under SAP Cloud SDK umbrella into `@sap-cloud-sdk` to the [npmjs.com](https://www.npmjs.com/).
+
+Most modules of the SAP Cloud SDK for JavaScript are Open Source released under Apache 2.0 license. It's preferred, but not mandatory that contribution happens under same conditions.
```

Comment:
I am no expert on this. But if we publish something not under a open source licence I guess this could lead to problems or irritate our user base:
https://docs.npmjs.com/files/package.json#license
I would suggest that contributors must use a licence from this list: https://opensource.org/licenses/alphabetical.

Reply:
I think we need to filter this list by Apache-2.0 compatible licenses...

Reply:
Let's ask legal :-)

Reply:
@marikaner can you take a led on this section?